tests/ext/test_updater.py

```diff
@@ -40,6 +39,8 @@
 from tests.auxil.slots import mro_slots
 
 if TEST_WITH_OPT_DEPS:
+    from tornado.netutil import bind_unix_socket
```

Comment:
this doesn't seem to be available on windows :/

Reply:
right, im blind. But now I can see two birds (codecov complaining) one stone, I would now do what we do in the handler
```
try:
    from tornado.netutil import bind_unix_socket

    UNIX_AVAILABLE = True
except ImportError:
    UNIX_AVAILABLE = False
```

and would do a new test class with @pytest.mark.skipif( and check if the error appears.

Do you see a better solution or is that fine

Reply:
You could also make the parametrization of the `unix` viariable in the tests dependent on `UNIX_AVAILABLE` - that would safe you from copying the whole test. I don't see a codecov complaind 🤔

Reply:
I saw this as a fail, apparently it succeeded after that.

![image](https://github.com/python-telegram-bot/python-telegram-bot/assets/25934244/2f581878-07dc-4354-bea2-5f5f5324b57e)

Reply:
if e.g. only windows tests ran so far, then the coverage is not full yet, as coverage results will be added as the tests complete :)